clang/test/CodeGenSYCL/anonymous_integration_footer.cpp

```diff
@@ -14,7 +14,8 @@ using namespace cl;
 // variable.
 struct S1 {
   static constexpr sycl::specialization_id a{1};
-  // CHECK: __SYCL_INLINE_NAMESPACE(cl) {
+  // CHECK: #include <CL/sycl/detail/defines_elementary.hpp>
```

Comment:
I'd rather this line here be moved 'above' these structs (perhaps above using namespace cl?) so it doesn't get lost/looks different from being associated with this spec-id.

Reply:
Fixed in e56ee417374054e0bcccc8574192cf51601722cf